pkg/apb/client.go

```diff
@@ -56,52 +53,23 @@ type Client struct {
 	log           *logging.Logger
 }
 
-func createClientConfigFromFile(configPath string) (*restclient.Config, error) {
-	clientConfig, err := clientcmd.LoadFromFile(configPath)
-	if err != nil {
-		return nil, err
-	}
-
-	config, err := clientcmd.NewDefaultClientConfig(*clientConfig, &clientcmd.ConfigOverrides{}).ClientConfig()
-	if err != nil {
-		return nil, err
-	}
-	return config, nil
-}
-
 func NewClient(log *logging.Logger) (*Client, error) {
```

Comment:
@eriknelson I agree, but this is a much larger change since Client is a class.  So things like RunImage need to be distributed out of client.go.  For now, I'm going to keep this in place, but adding a TODO in the final patch for this series and we can return to it later.